sphinxcontrib/openapi/openapi20.py

```diff
@@ -104,7 +104,7 @@ def _convert(schema, name='', required=False):
 
         type_ = schema.get('type', 'any')
         required_properties = schema.get('required', ())
-        if type_ == 'object':
+        if type_ == 'object' and schema.get('properties'):
```

Comment:
I don't quite understand what it fixes. The line below safely retrieves properties (if any) and will be no-op if there are no properties. Unless I'm missing something, it seems to be unnecessary check. Can you please explain this change, @tuffnatty?

Reply:
@ikalnytskyi It would be a no-op if there were no `else` part. The change makes an `object` without `properties` output something, while before the change, an `object` without `properties` would output nothing.